extensions/cornerstone-dicom-seg/src/panels/Toolbox/ThresholdSettingsPreset.tsx

```diff
@@ -0,0 +1,88 @@
+import React, { useState } from 'react';
+import { InputDoubleRange, Select } from '@ohif/ui';
+
+const defaultOptions = [
+  {
+    value: 'Soft tissue',
+    label: 'Soft tissue',
+    range: [-160, 240] as [number, number],
+  },
+  {
+    value: 'Lung',
+    label: 'Lung',
+    range: [-1350, 150] as [number, number],
+  },
+  {
+    value: 'Liver',
+    label: 'Liver',
+    range: [15, 165] as [number, number],
+  },
+  {
+    value: 'Bone',
+    label: 'Bone',
+    range: [-770, 1730] as [number, number],
+  },
+  {
+    value: 'Brain',
+    label: 'Brain',
+    range: [0, 80] as [number, number],
+  },
+];
+
+function ThresholdSettings({ onRangeChange }) {
+  const [options, setOptions] = useState(defaultOptions);
+  const [selectedPreset, setSelectedPreset] = useState(defaultOptions[0].value);
+
+  const handleRangeChange = newRange => {
+    const selectedOption = options.find(o => o.value === selectedPreset);
+
+    if (newRange[0] === selectedOption.range[0] && newRange[1] === selectedOption.range[1]) {
+      return;
+    }
+
+    onRangeChange(newRange);
+
+    const updatedOptions = options.map(o => {
```

Comment:
Hmmm... I believe it would be safest to do this as part of the update function that can be passed to `setOptions`?

```
setOptions((options) => {
  return options.map(...)
});
```

Reply:
I forgot to remove this file, it is outdated and we don't need it